cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/SecureStoreHandler.java

```diff
@@ -63,26 +64,23 @@ public SecureStoreHandler(SecureStore secureStore, SecureStoreManager secureStor
   public void create(HttpRequest httpRequest, HttpResponder httpResponder, @PathParam("namespace-id") String namespace,
                      @PathParam("key-name") String name) throws BadRequestException, IOException {
     if (!EntityId.isValidStoreKey(name)) {
```

Comment:
this should probably move to an entityid class for secure keys. Can you add a todo here, for handling when we add such a class?